cmd/autoheal/healer.go

```diff
@@ -80,38 +73,44 @@ func NewHealerBuilder() *HealerBuilder {
 	return b
 }
 
-// KubernetesClient sets the Kubernetes client that will be used by the healer.
+// ConfigFile adds one configuration file.
 //
-func (b *HealerBuilder) KubernetesClient(client kubernetes.Interface) *HealerBuilder {
-	b.k8sClient = client
+func (b *HealerBuilder) ConfigFile(path string) *HealerBuilder {
+	b.configFiles = append(b.configFiles, path)
 	return b
 }
 
-// OpenShiftClient sets the OpenShift client that will be used by the healer.
-//
-func (b *HealerBuilder) OpenShiftClient(client openshift.Interface) *HealerBuilder {
-	b.osClient = client
-	return b
-}
-
-// InformerFactory sets the OpenShift informer factory that will be used by the healer.
+// KubernetesClient sets the Kubernetes client that will be used by the healer.
 //
-func (b *HealerBuilder) InformerFactory(factory genericinf.SharedInformerFactory) *HealerBuilder {
-	b.informerFactory = factory
+func (b *HealerBuilder) KubernetesClient(client kubernetes.Interface) *HealerBuilder {
+	b.k8sClient = client
 	return b
 }
 
 // Build creates the healer using the configuration stored in the builder.
 //
 func (b *HealerBuilder) Build() (h *Healer, err error) {
+	// Load the configuration files:
+	if b.configFiles == nil || len(b.configFiles) == 0 {
```

Comment:
can be simplified to `len(b.configFiles) == 0` since [len of a nil object is 0](https://golang.org/pkg/builtin/#len)

Reply:
Done.